R/notes_to_fig_alt.R

```diff
@@ -0,0 +1,257 @@
+#' Download Google Slides pptx file
+#'
+#' @param id Identifier of Google slides presentation, passed to
+#' \code{\link{get_slide_id}}
+#'
+#' @note This downloads presentations if they are public and also try to make
+#' sure it does not fail on large files
+#' @return Downloaded file (in temporary directory)
+#' @export
+get_gs_pptx <- function(id) {
```

Comment:
Ideally we would just download one slide's powerpoint (which would contain its slide notes as well). But I haven't found how to do this.